diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4e7f14a..a12cba8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -372,6 +372,7 @@ endif()
 if(NOT (UNIX AND APPLE))
     # Install "portable.txt" if selected component is "portable"
     install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/${Launcher_Portable_File}" DESTINATION "." COMPONENT portable EXCLUDE_FROM_ALL)
+    install(CODE "file(MAKE_DIRECTORY \${CMAKE_INSTALL_PREFIX}/UserData)" COMPONENT portable EXCLUDE_FROM_ALL)
 endif()
 
 if(UNIX AND APPLE)
@@ -453,6 +454,8 @@ elseif(UNIX)
     # Install basic runner script if component "portable" is selected
     configure_file(launcher/Launcher.in "${CMAKE_CURRENT_BINARY_DIR}/LauncherScript" @ONLY)
     install(PROGRAMS "${CMAKE_CURRENT_BINARY_DIR}/LauncherScript" DESTINATION "." RENAME ${Launcher_Name} COMPONENT portable EXCLUDE_FROM_ALL)
+    install(CODE "file(MAKE_DIRECTORY \${CMAKE_INSTALL_PREFIX}/UserData)" COMPONENT portable EXCLUDE_FROM_ALL)
+
 
 elseif(WIN32)
     set(BINARY_DEST_DIR ".")
